### ISSUE TITLE
Docker Dashboard not list all running Containers/Apps

### ISSUE THREAD

**Issue by abhinav-chittora** [docker/for-mac#5016]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: C8CF5302-CA7C-4DF0-913E-38E66FED3742/20201022044614

### Expected behavior
When you open **Dashboard**, using menu the window that opens with **Containers/App** tabs, should list all running containers. 
### Actual behavior
In my case, I can only see containers which I started after I open **Dashboard**. Any container which was running before opening the dashboard is not listed in Dashboard -> Container/App tab. 
### Information

  - No idea whether the problem is reproducible or started after upgrade. 
  - macOS Version: 10.15.7

### Diagnostic logs

```
Docker for Mac: version 2.4.0.0

```

### Steps to reproduce the behavior

  1. Run some container using docker run command. 
  2. Open Dashboard using menu and check running container list.

**Comment by soupli** [comment]
Closing the window and reopening Docker Desktop will "solve" this

**Comment by abhinav-chittora** [comment]
Have tried closing and reopening the window. Sometimes it works and sometimes it doesn't. Should it be persistent no matter what?

**Comment by soupli** [comment]
Yes. But this particularly seem to happen when you click on the system tray
icon > Dashboard in a fast manner.

Op vr 23 okt. 2020 11:30 schreef Abhinav Chittora <notifications@github.com

**Comment by mieubrisse** [comment]
Adding a +1 that I've been experiencing the same issue for the past several months (v3.0.3 and previous)
